src/containerapp/HISTORY.rst

```diff
@@ -7,6 +7,12 @@ Upcoming
 * 'az containerapp env update': Fix bugs in update environment.
```

Comment:
```suggestion
* 'az containerapp env update': fix bugs in update environment.
```

Reply:
This comment fixed in
https://github.com/Azure/azure-cli-extensions/pull/6074